### ISSUE TITLE
"Create new post" does not show form

### ISSUE THREAD

**Issue by olleharstedt** [lelesys/Lelesys.Plugin.Forum#1]
Hi!

When clicking on the link "Create new post" no form is shown. This is our link:

`http://example.com/en/forum-page/fcat-1/forum.html?postPlugin=1&--lelesys_plugin_forum-forum[forum]=ec5d88f7-1ec6-dbd0-18c1-d17d32e77ba9`

Is anything missing? The page just says

`No Posts created for the forum`

Regards
Olle
Bitmotion

**Comment by olleharstedt** [comment]
One solution is to replace all `forum.identifier` in the templates with `forum.path`. The property mapper have problems with `identifier` for some reason.

**Comment by budhesh-naik** [comment]
Yes, Because {node.identifier} creates problem for property mapping in version 1.1.0 and above so we should use {node.path}.

**Comment by olleharstedt** [comment]
Btw, why do you handcraft the link to "Create new post" instead of using a `<f:link.action>...`?
